app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -36,6 +36,7 @@ public function panel(Panel $panel): Panel
             ->default()
             ->id('admin')
             ->path('admin')
+            ->topNavigation(true)
```

Comment:
Because of the badges not all of the labels of the navigation sidebar where visible anymore. So i added this line which changes the sidebar to a topbar: 
![image](https://github.com/pelican-dev/panel/assets/62171904/f9099cad-0c9a-4523-9a7f-cc0f83ea0975)

Should you not want this simply set it to false and then it is disables again. This can be added to a settings page which i can make if you guys want to.